apps/ledger-live-desktop/README.md

```diff
@@ -209,6 +209,6 @@ This part of the repository is where you will add the support of your blockchain
 For a smooth and quick integration:
 
 - See the developers’ documentation on the [Developer Portal](https://developers.ledger.com/docs/coin/general-process/) and
-- Go on Discord to chat with developer support and the developer community. See you there! If you are new to Ledger OP3N Discord server [click here](https://discord.gg/Ledger), otherwise directly join [the Blockchain channel](https://discord.com/channels/885256081289379850/907623688759803935).
+- Go on [Discord]((https://developers.ledger.com/discord-pro/)) to chat with developper support and the developper community.
```

Comment:
I think there is a single "p" on developer ! (to apply on other changes too)

Reply:
Yes, you're right! I've corrected everything.